recipes/squeegee/meta.yaml

```
@@ -0,0 +1,30 @@
+{% set version = "0.1.2" %}
+package:
+  name: squeegee
+  version: '{{version}}'
+source:
+  url: 'https://gitlab.com/treangenlab/squeegee/-/archive/v0.1.2/squeegee-v0.1.2.tar.gz'

```

To facilitate future updates, try this:
```suggestion
  url: 'https://gitlab.com/treangenlab/squeegee/-/archive/v{{ version }}/squeegee-v{{ version }}.tar.gz'
```